dts/bindings/clock/st,stm32u0-pll-clock.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2024 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  STM32U0 Main PLL node binding:
+
+  Takes one of clk_hse, clk_hsi or clk_msi as input clock, with an
+  input frequency from 4 to 16 MHz.
```

Comment:
Not really relevant in reality, but I'm curious why you are defining a higher value (4MHz) than the 2.66MHz stated in the reference manual?
And could you eventually rephrase the text a bit, to make it more clear that this is the frequency after the M divider.

Reply:
Yes , its right the input frequency from 2.66MHZ